templates/vagrantfiles/Vagrantfile.erb.erb

```diff
@@ -1,10 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
+#
+#
+# Usage:  vagrant init --template THIS_TEMPLATE boxname 
+
 
 Vagrant.configure("2") do |config|
-  c.ssh.insert_key = false
-  c.vm.define 'puppetserver' do |v|
-    v.vm.box = "<%%= box_name %>"
+  config.ssh.insert_key = false
+  config.vm.define 'puppetserver' do |v|
+    v.vm.box = "<%= @name %>"
```

Comment:
What's up with the double `%` in the rest of the file and the single one here?

Also, is this really an ERB that's generating another ERB?

Reply:
Yep, and [that's why the double `%%` is needed|https://ruby-doc.org/stdlib-2.1.9/libdoc/erb/rdoc/ERB.html#class-ERB-label-Recognized+Tags]—documentation coming up...

Reply:
Update: :ballot_box_with_check: